certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderAddress": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "full": {
+          "description": "Gets or Sets Full",
+          "type": "string"
+        },
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddressType"
+        },
+        "line1": {
+          "description": "Gets or Sets Line1",
+          "type": "string"
+        },
+        "line2": {
+          "description": "Gets or Sets Line2",
+          "type": "string"
+        },
+        "city": {
+          "description": "Gets or Sets City",
+          "type": "string"
+        },
+        "state": {
+          "description": "Gets or Sets State",
+          "type": "string"
+        },
+        "zip": {
+          "description": "Gets or Sets Zip",
+          "type": "string"
+        },
+        "county": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.County"
+        },
+        "country": {
+          "description": "Gets or Sets Country",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.CommunicationType": {     
+      "description": "Gets or Sets CommunicationType",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.Communication": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.CommunicationType"
+        },
+        "value": {
+          "description": "Gets or Sets Value",
+          "type": "string"
+        },
+        "primary": {
+          "description": "Gets or Sets Primary",
+          "type": "boolean"
+        },
+        "foreign": {
+          "description": "Gets or Sets Foreign",
+          "type": "boolean"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Irs1099": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "double",
+          "description": "Gets or Sets Id",
+          "type": "number"
+        },
+        "propertyOrServicesReceived": {
+          "description": "Gets or Sets PropertyOrServicesReceived",
+          "type": "string"
+        },
+        "overrideAutoProceedDistribution": {
+          "description": "Gets or Sets OverrideAutoProceedDistribution",
+          "type": "boolean"
+        },
+        "generate1099": {
+          "description": "Gets or Sets Generate1099",
+          "type": "boolean"
+        },
+        "allocationPercent": {
+          "format": "double",
+          "description": "Gets or Sets AllocationPercent",
+          "type": "number"
+        },
+        "allocationAmount": {
+          "format": "double",
+          "description": "Gets or Sets AllocationAmount",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Seller": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.